pmd_django/management/commands/dev.py

```diff
@@ -30,8 +39,15 @@ def serve():
                 raise CommandError("Unable to build frontend")
 
         try:
+            server_command = None
+            with open("package.json") as f:
+                if "\"main\"" in f.read():
```

Comment:
you know if python ternaries weren't so backwards this would be cleaner as a ternary 😂